packages/tailwind/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@vuestic/tailwind",
+  "description": "Script for config syncing between Tailwind CSS and Vuestic UI.",
+  "keywords": [
+    "vuesticui",
+    "vuestic",
+    "tailwindcss",
+    "tailwind",
+    "uikit",
+    "uiframework"
+  ],
+  "version": "0.1.2",
+  "license": "MIT",
+  "main": "./dist/sync.cjs",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "vite build"
+  },
+  "peerDependencies": {
+    "tailwindcss": "^3.2.4"
```

Comment:
We can probably add vuestic here as well.

Reply:
This makes vite to use `cjs`, better to use monorepo vuestic-ui in build.